src/components/Form.js

```diff
@@ -256,13 +377,62 @@ const StyledCategory = styled.p`
   }
 `;
 
+const StyledLabels = styled.label`
+  padding: 0 30px;
+`;
+
+const StyledInputs = styled.input`
+  background: transparent;
+  border: 1px solid rgba(71, 39, 35, 0.42);
+  border-radius: 5px;
+  padding: 1px;
+  width: 100%;
+  color: rgba(71, 39, 35, 0.72);
+  font-size: 20px;
+`;
+
 const StyledDate = styled.input`
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
     Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
   height: 30px;
+  background: transparent;
+  border: 1px solid rgba(71, 39, 35, 0.42);
+  border-radius: 5px;
+  padding: 1px;
+  width: 100%;
+  color: rgba(71, 39, 35, 0.72);
```

Comment:
here is the code the same like the StyledInput. You could use this as base for your StyledDate: 
const StyledDate = styled(StyledInput)`
...here the other stylings which are different...
`